src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -34,28 +33,21 @@ private static void checkJavaVersion() {
     final boolean v14 = version.contains("1.4");
     final boolean v15 = version.contains("1.5");
     if (v15 || v14 || v13 || v12) {
-      if (isMac()) {
-        JOptionPane.showMessageDialog(null,
-            "TripleA requires a java runtime greater than or equal to 6 [ie: Java 6]. (Note, this requires Mac OS X >= 10.5 or 10.6 depending.) \nPlease download a newer version of java from http://www.java.com/",
-            "ERROR", JOptionPane.ERROR_MESSAGE);
-        System.exit(-1);
-      } else {
-        JOptionPane.showMessageDialog(null,
-            "TripleA requires a java runtime greater than or equal to 6 [ie: Java 6]. \nPlease download a newer version of java from http://www.java.com/",
-            "ERROR", JOptionPane.ERROR_MESSAGE);
-        System.exit(-1);
-      }
+      JOptionPane.showMessageDialog(null,
+          "TripleA requires a java runtime greater than or equal to 8 [ie: Java 8]. \nPlease download a newer version of java from http://www.java.com/",
+          "ERROR", JOptionPane.ERROR_MESSAGE);
+      System.exit(-1);
     }
-  }// end checkJavaVersion()
+  }
 
   public static void main(final String[] args) {
+    PerfTimer timer = PerfTimer.startTimer("GameRunner1 Total Launch");
```

Comment:
IMO, let's remove any usages of PerfTimer for the moment. Keep things simpler, and it does seem to me that it looks like this timer is best for spot checking, or on-the-spot performance investigation. If this were server side code of some sort, we would want it, but since we have a thick client, unless we are going to report these values back to a centralized place, it is not necessary.